src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -280,22 +280,24 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		exposureSubmissionService.reset()
 
 		// Reset key value store. Preserve environment settings.
-		let environment = store.selectedServerEnvironment
+
 		do {
 			/// ppac API Token is excluded from the reset
 			/// read value from the current store
 			let ppacAPIToken = store.ppacApiToken
+			let environment = store.selectedServerEnvironment
 
 			let newKey = try KeychainHelper().generateDatabaseKey()
 			store.clearAll(key: newKey)
 
 			/// write excluded value back to the 'new' store
 			store.ppacApiToken = ppacAPIToken
-			store.lastAppReset = Date()
+			store.selectedServerEnvironment = environment
+            Analytics.collect(.submissionMetadata(.lastAppReset(Date())))
```

Comment:
Maybe it's just Github, but:
```suggestion
			store.selectedServerEnvironment = environment
                      Analytics.collect(.submissionMetadata(.lastAppReset(Date())))
```

Reply:
jep, only in Github :/